week02/Implement strStr.cpp

```diff
@@ -0,0 +1,47 @@
+class Solution {
```

Comment:
Where is the complexity analysis?

Reply:
> Where is the complexity analysis?

sorry vaia forgot it...can I put it in the comments?

Reply:
No. You need to put your changes in your `submissions_week02` branch. The changes will appear here.